src/main/java/cn/nukkit/item/randomitem/Fishing.java

```diff
@@ -65,8 +66,6 @@ public static Item getFishingResult(int fortuneLevel, int lureLevel) {
     }
 
     private static float limitRange(float min, float max, float value) {
-        if (value >= max) return max;
-        if (value <= min) return min;
-        return value;
+        return NukkitMath.clamp(value, min, max);
```

Comment:
I think this method should be removed completely since it's just calling the clamp